packages/react/emotion-utils/src/FullHeight.spec.tsx

```diff
@@ -0,0 +1,31 @@
+import { render, screen } from '@testing-library/react';
+import { FullHeight } from './FullHeight';
+
+const setInnerHeight = (height: number) => {
+  Object.defineProperty(window, 'innerHeight', { writable: true, configurable: true, value: height });
+};
+
+describe('FullHeight', () => {
+  it('should have height 768px', () => {
+    render(<FullHeight>FullHeight</FullHeight>);
+
+    const fullHeightBox = screen.getByText('FullHeight');
```

Comment:
how about seperate FullHeight text? 
It's not directly related to the test, but it's a means of testing.
and it can sustain SSOT

```tsx

const CHILDREN_TEXT = 'FullHeight'; // I'd like to change the variable name to something else


describe('FullHeight', () => {
  it('should have height 768px', () => {
    render(<FullHeight>{CHILDREN_TEXT}</FullHeight>);

    const fullHeightBox = screen.getByText(CHILDREN_TEXT);
```

Reply:
@okinawaa 
Thank you for your feedback.
Your comments resonate with us. I will fix the code.